src/components/ha-tabs.ts

```diff
@@ -1,16 +1,28 @@
 import "@polymer/paper-tabs";
+import { customElement, property } from "lit-element";
+import { PaperTabsElement } from "@polymer/paper-tabs/paper-tabs";
+import { Constructor } from "../types";
+
+const PaperTabs = customElements.get("paper-tabs") as Constructor<
+  PaperTabsElement
+>;
 
-const PaperTabsClass = customElements.get("paper-tabs");
 let subTemplate;
 
-class HaTabs extends PaperTabsClass {
+@customElement("ha-tabs")
+export class HaTabs extends PaperTabs {
+  @property({ type: Number }) public firstTabWidth = 0;
+
+  @property({ type: Number }) public lastTabWidth = 0;
+
+  @property({ type: Boolean }) public _lastLeftHiddenState = false;
```

Comment:
The should be `private` and `internalProperty` also prepend private properties with an `_`

Reply:
But... you can't use properties in Polymer elements, this is for LitElements.